subprojects/emcee/queue-api-models/src/main/kotlin/com/avito/emcee/queue/Bucket.kt

```diff
@@ -6,4 +6,6 @@ import com.squareup.moshi.JsonClass
 public data class Bucket(
     val bucketId: String,
     val payloadContainer: PayloadContainer,
+    val analyticsConfiguration: Any,
```

Comment:
Why can't it be typed?

Reply:
We don't use this field yet, but Moshi requires to declare it. It will be typed later.